build/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Just wondering:
build is a directory that gradle generates also and we tend to ignore it, is this a convention or can it be anything?